graylog2-server/src/main/java/org/graylog2/shared/users/UserManagementService.java

```diff
@@ -18,6 +18,9 @@
 
 import org.graylog2.plugin.database.ValidationException;
 import org.graylog2.plugin.database.users.User;
+import org.graylog2.rest.models.users.requests.ChangeUserRequest;
```

Comment:
It feels wrong, that we leak an object from the REST API layer into the service layer. I think we should either create an additional DTO to transport the change request or we could simply move the `ChangeUserRequest` class from the `org.graylog2.rest.models.users.requests` package to `org.graylog2.shared.users`.